src/actions/project-text-search.js

```diff
@@ -45,6 +53,9 @@ export function closeProjectSearch() {
 
 export function searchSources(query: string) {
   return async ({ dispatch, getState }: ThunkArgs) => {
+    if (!query) {
```

Comment:
wondering if I should move this guard logic to the component instead?

Reply:
Probably a good idea.